modules/apps/analytics/analytics-client-js/src/utils/constants.js

```diff
@@ -14,7 +14,7 @@
 
 // AC Version
 
-export const ANALYTICS_CLIENT_VERSION = '1.0.1';
```

Comment:
FYI I didn't need to update the client version to the bug we fix

Reply:
O I updated it because of the new release version.